vars/buildFailureMessage.groovy

```diff
@@ -0,0 +1,17 @@
+@NonCPS
+def call(){
+    def ERROR_STRING = "ERROR"
+    def message = ""
```

Comment:
Let's use definite datatypes. We have spent a lot of time fixing that for Python

Reply:
Pushed in my latest commit